src/picobox/_stack.py

```diff
@@ -55,19 +55,26 @@ def do(magic):
             with picobox.push(barbox):
                 assert do() == 14
             assert do() == 43
+
+    :param box: A :class:`Box` instance to push to the top of the stack.
+    :param chain: (optional) Look up missed keys down the stack if ``True``.
```

Comment:
nit: maybe you should explicitly mention that the keys are looked up only one level up the stack. So to look up through multiple levels, each level must be created with `chain=True`